ups/sphgeom.table

```diff
@@ -1,2 +1,7 @@
+setupRequired(base)
+setupRequired(sconsUtils)
+setupOptional(doxygen)
+
 envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
 envPrepend(DYLD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
+envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
```

Comment:
I need it for any stack package to work on OS X. See http://dmtn-001.lsst.io for more details. All other packages have received this change.